docker/Dockerfile

```diff
@@ -91,5 +91,8 @@ RUN \
 USER node
 ENV NODE_ENV=production
 
+# Expose 4041 for NORTH PORT, 7896 for HTTP PORT
+EXPOSE ${IOTA_NORTH_PORT} ${IOTA_HTTP_PORT}
```

Comment:
This is missing the default values- currently the ports won't work without `ENV`s being passed it should read:

> 
> ```dockerfile
> # Expose IOTA_NORTH_PORT and  IOTA_HTTP_PORT
> EXPOSE ${IOTA_NORTH_PORT:-4041} ${IOTA_HTTP_PORT:-7896}
> ```

Reply:
Fixed by https://github.com/telefonicaid/iotagent-ul/pull/342/commits/02dd708ad494d1a8fd96dbfb1a43a0f107927330

Reply:
LGTM